main.py

```diff
@@ -129,6 +129,7 @@ def load_game(self):
         self.sprites.remove(self.message)
 
     def run(self):
+        pygame.display.init()
```

Comment:
Why is there a need for using `pygame.display.init()` twice in the same function? What purpose does it solve?